webapp/src/main/java/com/box/l10n/mojito/android/strings/AndroidStringDocumentWriter.java

```diff
@@ -56,6 +56,7 @@ private <W extends Writer> W buildWriter(W writer) throws TransformerException {
     transformer.setOutputProperty(OutputKeys.INDENT, "yes");
     transformer.setOutputProperty(OutputKeys.METHOD, "xml");
     transformer.setOutputProperty(OutputKeys.ENCODING, "UTF-8");
+    transformer.setOutputProperty("{http://xml.apache.org/xslt}indent-amount", "0");
```

Comment:
In java 8 the default indent-amount is 0, but for java 11 the default value is 4, which without this change makes unit tests to fail.